app/controllers/saml_sessions_controller.rb

```diff
@@ -1,8 +1,9 @@
 # require "ruby-saml"
 
 class SamlSessionsController < Devise::SamlSessionsController
+  skip_before_action :verify_authenticity_token, raise: false
   after_action :set_saml_session_index_on_session, only: :create
-  prepend_before_action :store_info_for_sp_initiated_logout, only: :destroy
+  prepend_before_action :verify_signed_out_user, :store_info_for_sp_initiated_logout, only: :destroy
```

Comment:
Not sure this this line needs to be here. Looks like it's already in the parent class:
https://github.com/apokalipto/devise_saml_authenticatable/blob/617e7904963f68d214436244cbe0adf1d9c307f0/app/controllers/devise/saml_sessions_controller.rb

Reply:
I agree, but what I have found in the past is that for some reason the callbacks are not picked up from the parent controller and I needed to duplicate them in our overwritten file, so I did that here. You can see that with the deleted line:
prepend_before_action :store_info_for_sp_initiated_logout, only: :destroy

This comes from the parent class and was required in the previous version. As the parent file changed, I changed this one.